packages/amplify_authenticator/lib/src/screens/authenticator_screen.dart

```diff
@@ -66,11 +66,15 @@ class AuthenticatorScreen extends StatelessAuthenticatorComponent {
     final bool isDesktop =
         screenSize.width > AuthenticatorContainerConstants.landScapeView;
```

Comment:
I think we should add a min height here as well. If a phone in rotated, it switches to the "desktop" view, which probably isn't what we want.

```suggestion
    final bool isDesktop =
        screenSize.width > AuthenticatorContainerConstants.landScapeView &&
            screenSize.height > AuthenticatorContainerConstants.mediumWidth;
```

I know you didn't name this - but `isDesktop` implies we only want to use this layout on desktop devices. We want to use it on large tablets too. I'm not sure what a good name would be here. Maybe `isLargeLayout` or something. Or you could invert the conditions and name it `isMobileLayout`.

Reply:
You can also use `screenSize.longestSide` which seems to give a decent behavior for me

Reply:
Moved to isMobile and used `screenSize.shortSide` to good effect. Thanks for the suggestions.